<!DOCTYPE html>
<html>
<body>
<style>
h1 {text-align: center;}
h2 {text-align: center;}
h3 {text-align: center;}
p {text-align: center;}
</style>
<h1 style="font-family:arial">SWANS</h1>
<h2 style="font-family:arial">SWANS â€“ A comparative Single-entity Workflow ANalysiS pipeline
</h2>

<p style="font-family:arial">
ABSTRACT: Single-cell/nucleus RNA-sequencing (scnRNAseq) is used to examine transcription within individual cells, and can be used to investigate cellular heterogeneity, identify cell types, uncover cell lineages, and explore the impacts of different experimental conditions. A wide variety of tools are available for analyzing and exploring scnRNAseq data, and nearly all such tools have numerous functions, parameters, and options. Altering these parameters and options can have a drastic impact on downstream results, and there is a need for scnRNAseq workflows and pipelines that can generate results based on a variety of input parameters so that the user may compare results between different configurations. Here we present SWANS (Single-entity Workflow ANalysiS Pipeline), a single cell/nucleus RNA-seq analysis pipeline controlled by Snakemake. SWANS provides options for quality control tools and multiple modes of processing in Seurat, including normalization and integration methods. To communicate analysis results, an interactive report/Shiny app is used to display UMAP clustering schemas along with relevant differentially expressed genes. This allows the user to select a schema that is biologically plausible and representative of the tissue. When optimal parameters have been selected, the pipeline can rename clusters with user-provided annotations, find up/down-regulated genes within all clusters across multiple conditions, and use these results to find biological processes (e.g., pathway analysis) for each cluster. The pipeline allows automated, reproducible, and scalable data analysis that leverages R packages and Snakemake to execute computationally demanding commands in parallel. 
</p>

<h3 style="font-family:arial">Pipeline will be available soon, check back for updates.
</h3>

<p style="font-family:arial"></p>
Contact <a href="https://github.com/beigelk">Kat Beigel</a> (beigelk@chop.edu) for more information. 
</p>


</body>
</html>
